pm_icecon/land_spillover.py

```diff
@@ -0,0 +1,125 @@
+"""Routines which implement land spillover corrections to sea ice concentration fields.
+
+land_spillover.py
+"""
+
+import os
+
+import numpy as np
+from scipy.signal import convolve2d
+
+
+def read_adj123_file(
+    gridid='psn12.5',
+    xdim=608,
+    ydim=896,
+    anc_dir='/share/apps/amsr2-cdr/nasateam2_ancillary',
+    adj123_fn_='{anc_dir}/coastal_adj_diag123_{gridid}.dat',
+):
+    """Read the diagonal adjacency 123 file."""
+    coast_adj_fn = adj123_fn_.format(anc_dir=anc_dir, gridid=gridid)
+    assert os.path.isfile(coast_adj_fn)
+    adj123 = np.fromfile(coast_adj_fn, dtype=np.uint8).reshape(ydim, xdim)
+
+    return adj123
+
+
+def create_land90_conc_file(
```

Comment:
What is "land90"? What's "land90-conc file" (docstring below). Some better descriptions of what these terms refer to would be helpful!

Reply:
I've added a description in the docstring